### DISCUSSION TITLE
Line number indentation

### DISCUSSION THREAD

**Discussion by laerttt** [LazyVim/LazyVim#2654] [Q&A] [answered]
How do I change line number indentation so that the line number I am currently at stays a little to the left

this how I have it
![image](https://github.com/LazyVim/LazyVim/assets/101653303/395f4e3d-05c5-4595-8e9f-21535fbe4ed4)

this is how I want to make it
![image](https://github.com/LazyVim/LazyVim/assets/101653303/c367d7c0-a230-4628-81b4-89b61c554851)

**Comment by dpetka2001** [comment, marked as answer]
You can't do that without disabling the `statuscolumn` that LazyVim provides or write your own custom one (or you can take Lazyvim's and customize it to your own needs in your personal configuration). To disable Lazyvim's `statuscolumn`, just put `vim.opt.statuscolumn = ""` in your `/lua/config/options.lua`.

**Reply by laerttt** [reply]
Thank you, very much appreciated
